tests/webapp/api/test_jobs_api.py

```diff
@@ -229,6 +229,25 @@ def test_job_list_excluded(webapp, eleven_jobs_stored, sample_data,
     assert resp['results'][0]['job_guid'] == '9abb6f7d54a49d763c584926377f09835c5e1a32'
 
 
+def test_job_list_no_signatures(webapp, test_repository, test_sheriff):
+    """
+    test retrieving jobs in case of no signatures of exclusion
+    profile.
+    """
+    exclusion_profile = ExclusionProfile.objects.create(
+        name="no_profile",
+        is_default=False,
+        author=test_sheriff,
+    )
+    exclusion_profile.save()
```

Comment:
Django's `.create()` does a save too, so the save here can be skipped (it was only needed before since the object was being modified after the initial create). With the `exclusion_profile.save()` removed the `exclusion_profile` variable will be unused, so the `.create()` call can be adjusted to not have its return value assigned to anything.